_includes/tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/console-producer-keys.sh

```diff
@@ -0,0 +1,3 @@
+docker exec -i broker /usr/bin/kafka-console-producer --topic example-topic --broker-list broker:9092\
```

Comment:
@bbejeck @colinhicks do you think we should showcase AK CLI, e.g. `kafka-console-producer` and `kafka-console-consumer` or Confluent CLI, e.g. `confluent local produce` and `confluent local consume`?

Reply:
Don't know I could go either way.

Reply:
+1 for Confluent CLI in this case. In particular, removing the need to `exec` through a container clarifies these basic operations.

Reply:
do we have an official docker image for `confluent` cli?

Reply:
Not an "official" one, no.  Options:

* Build our own
* Use training's Docker image (but FYI it's packed with other things and is 2.5GB)

Reply:
My vote is to build our own image for `confluent cli`

Reply:
I'd be happy to help build this image

Reply:
lets do it! +1 for our image - lean and mean!

Reply:
Already looking at a go alpine image and maybe a multi-stage build if necessary, will try and make it as small as possible

Reply:
What should happen in this image if user tries to run confluent local ?

Reply:
i think it will just fail

Reply:
>I'd be happy to help build this image

@rspurgeon thanks! I'll take you up on that

Reply:
Until we have our own Confluent CLI docker image to keep moving forward with this tutorial and upcoming console producer/consumer tutorials we can either: 

- Stick with the AK version, and update the syntax once we have our image
- Use training's Docker image and make sure we have our own before doing a release

WDYT?

Reply:
I can have one built by tomorrow (5/6/2020)

Reply:
That's awesome! Thanks

Reply:
As discussed out of band of this PR, there are complications with building a docker image to allow for a run of confluent cli or the ak tools and installing the tools has it's own burden on users.  In addition, asking a user to run `docker run --it --rm confluentinc/confluent-cli -arg1 -arg 2` doesn't feel (IMO) any more streamlined than the `docker exec` solution we have here now.  Are we ok leaving this as is for now until another solution is found?

Reply:
Another option is to have the user open a shell on the container and exercise the tutorial that way:

```
docker-compose exec broker bash
root@broker:/# kafka-topics --create --topic example-topic --zookeeper zookeeper:2181 --replication-factor 1 --partitions 1
Created topic example-topic.
root@broker:/# kafka-console-consumer --topic example-topic --bootstrap-server broker:9092
the
lazy
fox
jumped over the brown cow
^CProcessed a total of 4 messages
```

This would require the student to open 2 termainals and `exec broker bash` into both of them; however, we require them to open two terminals now so maybe this wouldn't be too big of a burden.

```
docker-compose exec broker bash
root@broker:/# kafka-console-producer --topic example-topic --broker-list broker:9092
>the
>lazy
>fox
>jumped over the brown cow
>^Croot@broker:/# kafka-console-producer --topic example-topic --broker-list broker:9092
>how now
brown cow
all streams lead
to Kafka!>>>
```

Reply:
@rspurgeon good idea.  I tried this out and I like the approach.  There really is no extra burden on the user as the tutorial requires opening two windows anyway.

If you can run through this again and give a +1 then I can merge, and this will unblock the other queued up PRs

@ybyzek can you weigh in if you agree with this approach? Then I can update the remaining tutorials accordingly

Reply:
@bbejeck just to confirm, I think you're asking for feedback on the approach to drop into the container and leave the terminal open as such  (versus weighing in on Confluent CLI):

```
docker-compose exec broker bash
root@broker:/# 
```

Makes sense to me, agree.

Reply:
@ybyzek 
>I think you're asking for feedback on the approach to drop into the container and leave the terminal open

Yes, that's correct. 

>KT tutorials about Avro/Protobuf/etc will require dropping into the schema-registry container, not broker.

Ack and thanks for pointing that out.